web/tests/functional/diff_local_remote/test_diff_local_remote.py

```diff
@@ -207,14 +207,14 @@ def test_local_cmp_filter_unres_severity(self):
     def test_local_cmp_filter_unres_filepath(self):
         """Filter unresolved results by file path."""
         res = self.get_local_remote_diff(['--file', '*divide_zero.cpp'])
-        self.assertEqual(len(re.findall(r'divide_zero.cpp', res)), 4)
+        self.assertEqual(len(re.findall(r'divide_zero.cpp', res)), 5)
```

Comment:
Please add a comment here that in the output the file names are printed twice because of the summary.
It will be really hard to follow, how many reports are actually found and printed.